build-genesis.sh

```diff
@@ -23,7 +23,7 @@ if ! [ -f genesis.json ]; then
 fi
 starsd export-airdrop-snapshot uatom genesis.json snapshot.json
 starsd init testmoniker --chain-id $CHAIN_ID
-starsd prepare-genesis testnet $CHAIN_ID
+starsd prepare-genesis mainnet $CHAIN_ID
```

Comment:
Seems a bit weird, but I understand why this is. Probably fine.

Reply:
Yeah just to get the same params as mainnet.